src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -40,6 +30,6 @@ public void draw() {
                            + " sq.units, bottom base: " + bottomBase
                            + " " + unitOrUnits(bottomBase) + ", top base: " + topBase
                            + " " + unitOrUnits(topBase) + ", height: " + height
-                           + " " + unitOrUnits(height) + ", color: " + getColor());
+                           + " " + unitOrUnits(height) + ", color: " + getColor().getInfo());

```

Comment:
[use method name() to get string representation of ENUM](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)